nobrainer/models/autoencoder.py

```diff
@@ -0,0 +1,82 @@
+"""Model definition for Autoencoder.
+"""
+
+import tensorflow as tf
+from tensorflow.keras import layers, models
+import math
+
+def autoencoder(input_shape, encoding_dim=512, n_base_filters=16, batchnorm=True, batch_size=None, name='autoencoder'):
+	"""Instantiate Autoencoder Architecture.
+
+    Parameters
+    ----------
+    input_shape: list or tuple of four ints, the shape of the input data. Omit
+        the batch dimension, and include the number of channels. Currently, only squares and cubes supported.
+    encoding_dim: int, the dimensions of the encoding of the input data. This would translate
+    	to a latent code of dimensions encoding_dimx1.
+    n_base_filters: int, number of base filters the models first convolutional layer. The subsequent layers
+    	have n_filters which are multiples of n_base_filters.
+    batchnorm: bool, whether to use batch normalization in the network.
+    batch_size: int, number of samples in each batch. This must be set when
+        training on TPUs.
+    name: str, name to give to the resulting model object.
+
+    Returns
+    -------
+    Model object.
+    Encoding layer.
+    """
+
+	conv_kwds = {
```

Comment:
Are you using tabs or spaces here? If you're using tabs, can you convert everything to spaces?